paper-tabs.html

```diff
@@ -376,6 +378,28 @@
         }, 10);
       },
 
+      _isItem: function(node) {
```

Comment:
Can this maybe be a closure inside of onIronItemsChanged? It doesn't seem to be used anywhere else...